### PR TITLE
.\backend_core\Pipfile

### PR DESCRIPTION
**Pull request by tkajdro**
Staram się :)

### CODE REVIEW

**Review by pawelkonior — APPROVED** [review]
Good job!

File: backend_core/config/settings.py [hunk 1]
```diff
@@ -121,3 +123,9 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+REST_FRAMEWORK = {
+    'DEFAULT_PERMISSION_CLASSES': [
+        'rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnly'
```
Comment (martynawitkowska) [inline backend_core/config/settings.py]: better add AllowAny permission class

**Comment by bartoszbaran2** [comment]
well done

**Review by bartoszbaran2 — COMMENTED** [review]
well done

**Review by bartoszbaran2 — APPROVED** [review]

**Pull request closed without merging**